### ISSUE TITLE
[Feature Request] Option for select interface language

### ISSUE THREAD

**Issue by eddiezato** [YACReader/yacreader#17]
I delete .qm files for my locale. But it is not proper way for this situation.

**Comment by selmf** [comment]
Yes, a more comfortable way to do this would be better. Not sure how hard this is to implement, but I'll look into it.

**Comment by luisangelsm** [comment]
YACReader 10 includes support for this.
